swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=0):
+        super().__init__(category="Clothing",condition=condition)
```

Comment:
great job on using inheritance here. Another way you could save some time on your code is to just leave `condition` as is instead of` condition = condition.` Nothing major just fyi.